src/OSPSuite.UI/Views/Importer/ModalImporterView.cs

```diff
@@ -22,26 +22,21 @@ public override void InitializeResources()
 
       public void AttachPresenter(IModalImporterPresenter presenter)
       {
-
       }
 
       public void FillImporterPanel(IView view)
       {
          importerPanelControl.FillWith(view);
       }
 
-
       public void AttachImporterPresenter(IImporterPresenter presenter)
       {
-         presenter.OnTriggerImport += (s, d) =>
-         {
-            DialogResult = System.Windows.Forms.DialogResult.OK;
-         };
+         presenter.OnTriggerImport += (s, d) => { DialogResult = DialogResult.OK; };
```

Comment:
This is so weird. A view that knows about another presenter and reacts to event... why?

Reply:
Look comment above on what is the intention of the ModalImporterView. This is just presenting the DataImporterPresenter as a dialog and when it imports, it should close the dialog with OK message. Again, we could rethink this a little bit.

Reply:
a view should never know of another presenter. This breaks encapsulation. Too late for now